core/controllers/acl_decorators.py

```diff
@@ -2024,6 +2065,8 @@ def test_can_perform(self, **kwargs):
             UnauthorizedUserException. The user does not have
                 credentials to access the page.
         """
+        # The X-AppEngine-Cron header is set inside AppEngine and if a request
```

Comment:
Ditto, suggest adding link to the comment.

Reply:
Yes, this was tested and I added the link.